mycore-user2/src/main/java/org/mycore/user2/MCRUser.java

```diff
@@ -664,7 +668,7 @@ public void setValidUntil(Date validUntil) {
         return Arrays.stream(getRoles()).map(MCRRole::getName).collect(Collectors.toSet());
     }
 
-    @MCRUserAttribute(name = "roles", separator = ";")
+    @org.mycore.user2.annotation.MCRUserAttribute(name = "roles", separator = ";")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.mycore.user2.annotation.MCRUserAttribute' due to existing same package import 'org.mycore.user2.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=4797507)